dashboard/settings.py

```diff
@@ -119,6 +121,9 @@ def path(*parts):
 # https://docs.djangoproject.com/en/1.9/topics/i18n/
 
 LANGUAGE_CODE = 'en-us'
+LOCALE_PATHS = [
+    path('locale',),
```

Comment:
It's usually because python treats strings (like `'locale'`) as a list (of characters). In this case, passing just `'locale'` to `path` means that `path(*parts)` might treat `'locale'` as arguments `"l"`, `"o"`, `"c"`, `"a"`, `"l"`, and `"e"`. Adding a `,` to the end of the string passes it as a list: `['locale']` so that `*parts` expands it to just the single argument: `"locale"`.